crates/solver/src/settlement.rs

```diff
@@ -251,6 +256,23 @@ impl Settlement {
             .sum()
     }
 
+    // Computes the total scaled unsubsidized fee of all protocol trades (in wei ETH).
+    pub fn total_unscaled_subsidized_fees(
```

Comment:
nit: maybe add unit test for this function (noticed the decrease in unit test coverage)

Reply:
Decided against this because the functions are so similar didn't feel test was worth it.